IPython/core/tests/test_inputtransformer.py

```diff
@@ -57,13 +57,13 @@ def transform_checker(tests, transformer, **kwargs):
 
        assign_magic =
        [(i,py3compat.u_format(o)) for i,o in \
-       [(u'a =% who', "a = get_ipython().magic({u}'who')"),
-        (u'b = %who', "b = get_ipython().magic({u}'who')"),
-        (u'c= %ls', "c = get_ipython().magic({u}'ls')"),
+       [(u'a =% who', "a = get_ipython().run_line_magic({u}'who', {u}'')"),
```

Comment:
So long as we're targeting IPython 6.x, we can get rid of the `{u}` things in the tests - that's for Python 2 compatibility, which we've dropped now.